saleor/checkout/utils.py

```diff
@@ -713,3 +765,7 @@ def validate_variants_in_checkout_lines(lines: Iterable["CheckoutLineInfo"]):
                 )
             }
         )
+
+
+def get_base_money(taxed_money: "TaxedMoney", from_gross: bool) -> Money:
```

Comment:
Not sure if the name gives enough context. What is base_money? The function returns a money amount which we used in all calculations, based on the flag value